addons/l10n_ch/data/account_vat2011_data.xml

```diff
@@ -677,6 +677,26 @@
             <field name="chart_template_id" ref="l10nch_chart_template"/>
             <field name="type_tax_use">purchase</field>
             <field name="tax_group_id" ref="tax_group_tva_0"/>
+            <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                }),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+            <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                }),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
```

Comment:
Aren't repartition lines supposed to be created by default when you don't specify them ? :o (I'm nearly sure they are :/ Bug ?)

Reply:
I confirm it is not created by default. Worth to check why since it's in the `default_get` but not in this PR's scope.